test/xpu/xpu_test_utils.py

```diff
@@ -252,6 +252,10 @@
     "square",
     "heaviside",
     "argsort",
+    "special_bessel_j0",
+    "special_bessel_j1",
```

Comment:
Those operators are not in common_methods_invocations, don't need to add here. please remove.

Reply:
Thanks for pointing out. Will remove.